identity-apps-core/apps/authentication-portal/src/main/webapp/org_discovery.jsp

```diff
@@ -154,6 +154,10 @@
                      <input type="submit" id="orgNameButton" onclick="enterOrgName();" class="ui primary basic button link-button"
                            value="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "provide.organization.name")%>">
                   </div>
+                  <div class="social-login blurring social-dimmer">
+                     <input type="submit" id="orgHandleButton" onclick="enterOrgHandle();" class="ui primary basic button link-button"
+                           value="<%=AuthenticationEndpointUtil.i18n(resourceBundle, "provide.organization.handle")%>">
```

Comment:
Let's use the new i18n function[1] that supports custom text branding.

[1] https://github.com/wso2/identity-apps/blob/master/identity-apps-core/apps/authentication-portal/src/main/webapp/includes/localize.jsp#L174